internal/querynodev2/segments/segment_loader.go

```diff
@@ -243,18 +244,21 @@ func (loader *segmentLoader) Load(ctx context.Context,
 			return nil, errors.New("segment was removed from the loading map early")
 		}
 
+		log.Info("wait segment loaded...", zap.Int64("segmentID", segment.GetSegmentID()))
```

Comment:
Will there be too much info in the logs?

Reply:
won't, as the log is for load only, and there are few segments